l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -128,6 +128,10 @@ def getPartnerBase(self, DatiAnagrafici, supplier=True):
             partners = partner_model.search([
                 ('vat', '=', vat),
             ])
+            if len(partners) > 1:
+                partners = partners.filtered(lambda p: p.fiscalcode == cf)
+            if len(partners) > 1:
+                partners = partners.mapped('commercial_partner_id')
```

Comment:
Nella chat si parlava di fare una modifica del genere, senza nessun'altra modifica (aggiunta di campi vat_group ecc., essendo già logico di default che alla stessa p.iva corrisponde un gruppo, che poi se si vuole installare il modulo partner_affiliate per tenerne traccia lo si può fare) 
```suggestion
            domain = [('vat', '=', vat),]
            if cf:
                domain.append(('cf', '=', cf'))
            partners = partner_model.search()
```

Reply:
@sergiocorato ok, modificato